rspotify-model/src/track.rs

```diff
@@ -92,7 +92,7 @@ pub struct SavedTrack {
 }
 
 /// Track id with specific positions track in a playlist
-pub struct TrackPositions {
+pub struct ItemPositions {
```

Comment:
OK, so I've modified `ItemPositions` so that it uses borrowed data instead of owned, including `&dyn PlayableId` instead of `Box<dyn PlayableId>`. This is because `ItemPositions` is only passed as a short-lived wrapper and doesn't really need to hold data, in my opinion. This way we can avoid two unnecessary allocations. If that's OK with you you can merge. Otherwise we can make it owned.